src/lib/model-handler.ts

```diff
@@ -14,24 +15,28 @@ export class ModelHandler {
   private static handleModel(model: FormModel, group: FormGroup, builderInstance: FormBuilderComponent): void {
     Object.keys(model).forEach((name) => {
       const field = model[name];
-      if (field) {
-        const control = ModelHandler.buildSingleField(field, builderInstance);
-        if (!control) {
-          return;
-        }
-        control.setParent(group);
-
-        if (field.disabled) {
-          control.disable();
-        }
+      if (!field) {
+        return;
+      }
 
-        group.addControl(name, control);
-        if (field.builderInstance == null && builderInstance) {
-          field.builderInstance = builderInstance;
-        }
+      if (field instanceof AbstractLayoutType) {
+        this.handleModel(field.options.model, group, builderInstance);
+        return;
+      }
```

Comment:
If the field is a layout field just proxying the model in the same group.

For example if the model is
```typescript
const model = {
    row: new RowLayout({
        model: {firstName: new TextField()}
    }),
};
```

The model handler will create a form group like this
```typescript
const formGroup = new FormGroup({
    firstName: new FormControl(),
});
```